parachain/pallets/ethereum-beacon-client/src/benchmarking/mod.rs

```diff
@@ -86,11 +85,23 @@ benchmarks! {
 				block_update.block.slot,
 			), initial_sync_data.current_sync_committee);
 
+		let finalized_update: FinalizedHeaderUpdate<T::MaxSignatureSize, T::MaxProofBranchSize, T::MaxSyncCommitteeSize> = finalized_header_update();
+
+		let finalized_slot = finalized_update.finalized_header.slot;
+		let finalized_block_root: H256 =
+			merkleization::hash_tree_root_beacon_header(finalized_update.finalized_header)
+				.unwrap()
+				.into();
+
```

Comment:
The execution header import is now dependent on a child finalized header after it for the ancestry proof.